Tools/LambdaTestTool-v2/src/Amazon.Lambda.TestTool/Amazon.Lambda.TestTool.csproj

```diff
@@ -15,13 +15,17 @@
   </PropertyGroup>
 	
   <ItemGroup>
+      <PackageReference Include="Amazon.Lambda.RuntimeSupport" Version="1.11.0" />
       <PackageReference Include="Blazored.Modal" Version="7.3.1" />
 	  <PackageReference Include="Microsoft.Extensions.FileProviders.Embedded" Version="8.0.11" /> 
-  </ItemGroup>	
+  </ItemGroup>
+
+  <ItemGroup>
+	  <None Include="$(OutputPath)Amazon.Lambda.RuntimeSupport.dll" Pack="true" PackagePath="content" Visible="false" />
```

Comment:
i still need to test end to end with aspire poc if any more (lambda core dlls) are needed or anything else